src/datasets.c

```diff
@@ -593,11 +605,29 @@ void DatasetPostReloadCleanup(void)
     SCMutexUnlock(&sets_lock);
 }
 
+static void GetDefaultMemcap(uint64_t *memcap, uint32_t *hashsize)
+{
+    const char *str = NULL;
+    if (ConfGetValue("datasets.defaults.memcap", &str) == 1) {
+        if (ParseSizeStringU64(str, memcap) < 0) {
+            FatalError(SC_ERR_FATAL, "memcap value cannot be deduced: %s", str);
```

Comment:
Should we fail here or fallback to defaults in the code with a warning?

Reply:
Yes OK.

Reply:
Can we store the parsed values instead of re-parsing them every time?